test/Microsoft.AspNetCore.Authentication.Test/GoogleTests.cs

```diff
@@ -545,30 +545,70 @@ private void ConfigureDefaults(GoogleOptions o)
         }
 
         [Fact]
-        public async Task ChallengeWillUseAuthenticationPropertiesAsParameters()
```

Comment:
Keep the old google test to ensure no breaking change?  Do we want to add a test for each provider using AuthenticationProperties instead of the sugar ChallengeProperties to verify that things work still?

Reply:
Hm, I had that mixed test as a way to ensure that the old `Items` stuff still works, but I guess I can also keep the original test for that.

I’m not sure if we need tests for `AuthenticationProperties`; the `OAuthChallengePropertiesTests` should already cover that the subclasses are compatible. Although I guess a handler could do an explicit type check there… I’ll add tests 🤷‍♂️ 😄

Reply:
Done ✔️